docs/x-pack.rst

```diff
@@ -0,0 +1,76 @@
+Configuring X-Pack (Security)
+***********************
+
+X-Pack is the collection of extensions provided by elastic to enhance the capabilities of the Elastic Stack with things
+such as reporting, monitoring and also security. If you installed x-pack your cluster will now be protected with the
+security module, this will also be like this if you are using Elasticsearch through the Elastic Cloud solution.
+
+=====================================================
+Setup roles and users.
+=====================================================
+
+After installing the plugin, the first thing you will have to do is configure the necessary users and roles to access let
+the LTR plugin operate.
+
+We recommend you to create two separate roles, one for administrative task such as creating the models, updating the feature sets, etc ..
+and one to run the queries.
+
+For this configuration, we supose you already have identified, and created two users, one for running queries and one for doing administrative tasks. If you
+need help to create the users, we recommend you to check the `x-pack api documentation for user management <https://www.elastic.co/guide/en/elasticsearch/reference/6.1/security-api-users.html>`_.
+
+
+To create the two roles, you can do it with this commands,
+
+    POST /_xpack/security/role/ltr_admin
+    {
+        "cluster": [ "ltr" ],
+        "indices": [
+            {
+                "names": [ ".ltrstore*" ],
+                "privileges": [ "all" ],
+            }
+        ]
+    }
+
+    POST /_xpack/security/role/ltr_query
```

Comment:
Should we document that read access to models via the `sltr` query is not strictly gated by x-pack? The read access will only be checked if the model needs to be loaded. If the model is already in cache for that node no checks will be performed.  While it's not important to me i could imagine someone, somewhere, being surprised.

Reply:
Fair point to add in the docs. +1